recipes-browser/cog/cog/0001-Install-a-libWPEBackend-default.so-symlink-with-fdo.patch

```diff
@@ -0,0 +1,30 @@
+From ea6475e893705fbf5c2144f9f83bde370ce1a9e2 Mon Sep 17 00:00:00 2001
+From: Carlos Alberto Lopez Perez <clopez@igalia.com>
+Date: Tue, 3 Jul 2018 16:16:47 +0200
+Subject: [PATCH] Install a libWPEBackend-default.so symlink with fdo
+
```

Comment:
JFTR, this will be unneeded after https://github.com/Igalia/cog/issues/11 is solved — I'll be looking forward to dropping this kind of downstream patches in the future, but it's fine to have this here for the time being.